CHANGELOG.md

```diff
@@ -3,6 +3,86 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.6.0
+
+_Mar 3, 2022_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 📦 Use the same bundling scripts as the core repository (#3965) @flaviendelangle
```

Comment:
I think we should avoid to talk about "core" to talk about "the core team" on public content.

```suggestion
- 📦 Use the bundling scripts as the non-X packages (#3965) @flaviendelangle
```

Or

```suggestion
- 📦 Use the bundling scripts as the `@mui/material` package (#3965) @flaviendelangle
```

Or

```suggestion
- 📦 Use the bundling scripts as the packages published by the [https://github.com/mui/material-ui](material-ui) repository (#3965) @flaviendelangle
```

Reply:
I took the last option